js/jquery-comments.js

```diff
@@ -594,13 +594,23 @@
         },
 
         updateToggleAllButton: function(parentEl) {
+            // Don't hide replies if maxRepliesVisible has negative value
+            if (this.options.maxRepliesVisible < 0) return;
```

Comment:
Wouldn't this also return in case `this.options.maxRepliesVisible` equals `0`?

How about `if (this.options.maxRepliesVisible == null) return;`? According to [SO](http://stackoverflow.com/a/27757708) it should work for both `null` and `undefined`.